### PR TITLE
Sort robots by status and name

### PR DESCRIPTION
**Pull request by Eddasol**

### CODE REVIEW

File: frontend/src/components/Pages/FrontPage/RobotCards/RobotStatusSection.tsx [hunk 1]
```diff
@@ -35,7 +36,15 @@ export const RobotStatusSection = () => {
             (robot) =>
                 robot.currentInstallation.installationCode.toLocaleLowerCase() === installationCode.toLocaleLowerCase()
         )
-        .sort((robot, robotToCompareWith) => (robot.status! > robotToCompareWith.status! ? 1 : -1))
+        .sort((robot, robotToCompareWith) =>
+            robot.status! !== robotToCompareWith.status!
+                ? robot.status! > robotToCompareWith.status!
+                    ? 1
+                    : -1
```
Comment (tsundvoll) [inline frontend/src/components/Pages/FrontPage/RobotCards/RobotStatusSection.tsx:43]: 1 and -1  are kind of magic numbers. Do they represent True / False, could we use that instead?

**Review by tsundvoll — CHANGES_REQUESTED** [review]

**Review by UsamaEquinorAFK — APPROVED** [review]
lgtm but agree with Thomas  :p

File: frontend/src/components/Pages/FrontPage/RobotCards/RobotStatusSection.tsx (hunk 1 above)
Comment (Eddasol) [reply frontend/src/components/Pages/FrontPage/RobotCards/RobotStatusSection.tsx:43]: Kind of. Usually sorting is done by returning a - b but when sorting by something that isn't a number this is the common way to do it in typescript.

Comment (tsundvoll) [reply frontend/src/components/Pages/FrontPage/RobotCards/RobotStatusSection.tsx:43]: OKey, good clarification, thank you

**Review by tsundvoll — APPROVED** [review]

**Review by andchiind — APPROVED** [review]
LGTM

**Pull request merged**